.github/workflows/security.yml

```diff
@@ -0,0 +1,28 @@
+on: 
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - "**/*.md"
+  schedule:
+    - cron: 15 14 * * 6
+
+jobs:
+  veracode-sca-task:
+    runs-on: ubuntu-latest
+    name: Scan repository for Issues
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Run Veracode SCA
```

Comment:
Where are the results being stored?

Reply:
The results are created as Github issues.